quotest/quotest.cpp

```diff
@@ -517,6 +517,7 @@ TEST_IMPL(showLocalUsername)
         // it is using the id fallback :(
         FAIL_TEST();
     }
+    FINISH_TEST(true);
```

Comment:
You can actually just swallow the above condition inside `FINISH_TEST()`

Reply:
...and what if the test user's display name just not set, by the way?

Reply:
Upon some thinking - maybe we could make it a pre-requisite for the test user, to have a non-empty display name.

Reply:
I simplified the condition but I'm not sure how to make it a pre-requisite. It seems like renaming the user is currently the only way to fill `defaultName` with a correct value... So I can't really test if it's working.

Reply:
Change their displayname once out of band (literally login and change it), and after testing restore (in quotest) to a non-empty value?